### PR TITLE
arcconf Migration and rebuild specific tests - v2

### PR DESCRIPTION
**Pull request by manvanthar**
arcconf_migration.py tests Migration and rebuild specific tests.
includes the suggestion from v1 review.

Signed-off-by: Manvanthara B Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by manvanthar** [comment]
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/645422/job.txt)
[results.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/645423/results.txt)

File: io/disk/arcconf/arcconf_migration.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                self.repo = self.fetch_asset(http_repo, expire='10d')
+                cmd = "dpkg -i %s" % self.repo
+            else:
+                http_repo = "%s%s.rpm" % (self.http_path, self.tool_name)
+                self.repo = self.fetch_asset(http_repo, expire='10d')
+                cmd = "rpm -ivh %s" % self.repo
+            if process.system(cmd, ignore_status=True, shell=True) == 0:
+                self.skip("Unable to install arcconf")
+
+        self.os_drive = ""
```
Comment (narasimhan-v) [inline io/disk/arcconf/arcconf_migration.py]: This is not needed

File: io/disk/arcconf/arcconf_migration.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        drive = self.cmdop_list("OS")
+
+        if drive != "":
+            # Delete a logical drive only if it is not OS drive
+            cmd = "df -h /boot | grep %s" % drive
+            if process.system(cmd, timeout=300, ignore_status=True,
+                              shell=True) != 0:
+                cmd = "echo y | arcconf delete %s logicaldrive %s" % \
+                      (self.crtl_no, self.logicaldrive)
+                self.check_pass(cmd, "Failed to cleanup Logical drive")
```
Comment (narasimhan-v) [inline io/disk/arcconf/arcconf_migration.py]: The work done here is similar to lines 104-115, with slight differences. please consider a helper function here, if possible.

File: io/disk/arcconf/arcconf_migration.py (hunk 1 above)
Comment (manvanthar) [inline io/disk/arcconf/arcconf_migration.py]: done

File: io/disk/arcconf/arcconf_migration.py (hunk 2 above)
Comment (narasimhan-v) [inline io/disk/arcconf/arcconf_migration.py]: @manvanthar we can skip from helper function.

Comment (manvanthar) [inline io/disk/arcconf/arcconf_migration.py]: @narasimhan-v  tried your suggestion. the code seems to be running, the fix will be in v3

**Comment by manvanthar** [comment]
respinnning in v3

**Pull request closed without merging**